ci-operator/step-registry/ipi/conf/aws/custom-vpc/ipi-conf-aws-custom-vpc-commands.sh

```diff
@@ -10,6 +10,14 @@ CONFIG="${SHARED_DIR}/install-config.yaml"
 subnet_ids_file="${SHARED_DIR}/subnet_ids"
 az_file="${SHARED_DIR}/availability_zones"
 
+
+if [[ "${PUBLISH}" == "Internal" ]]; then
```

Comment:
@gpei is working on the issue in https://github.com/openshift/release/pull/61687